cfme/tests/control/test_actions.py

```diff
@@ -108,9 +109,10 @@ def vm(request, provider_mgmt, provider_crud, provider_key, provider_data, small
     if isinstance(provider_mgmt, mgmt_system.RHEVMSystem):
         # RHEV-M is sometimes overloaded, so a little protection here
         try:
-            provider_mgmt.deploy_template(
-                small_template,
-                vm_name=vm_name,
+            deploy_template(
```

Comment:
Ok, I will then update the deploy_template function in virtual_machines to take both.